Doc/conf.py

```diff
@@ -157,6 +157,77 @@
     ('envvar', 'USER'),
     ('envvar', 'USERNAME'),
     ('envvar', 'USERPROFILE'),
+]
+
+# Temporary undocumented names.
+# In future this list must be empty.
```

Comment:
A fix would be to replace ``:c:data:`Exception` `` with ``.. c:data:`Exception` `` in Doc/c-api/exceptions.rst, right?

Reply:
I tried this, and it turned out not so simple. You need to specify the type:
```
.. c:data:: PyObject *PyExc_SomeError
```
and the result does not look good.